pgx-utils/src/sql_entity_graph/pg_extern/entity/mod.rs

```diff
@@ -51,32 +50,6 @@ pub struct PgExternEntity {
     pub to_sql_config: ToSqlConfigEntity,
 }
 
-impl std::hash::Hash for PgExternEntity {
-    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
-        self.full_path.hash(state);
-    }
-}
-
-impl PartialEq for PgExternEntity {
-    fn eq(&self, other: &Self) -> bool {
-        self.full_path.eq(other.full_path)
-    }
-}
-
-impl Eq for PgExternEntity {}
-
-impl Ord for PgExternEntity {
-    fn cmp(&self, other: &Self) -> Ordering {
-        self.full_path.cmp(&other.full_path)
-    }
-}
-
-impl PartialOrd for PgExternEntity {
-    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
-        Some(self.cmp(other))
-    }
-}
-
```

Comment:
I see how these impls are broken, but I also see the intention here.

The intention, I believe, is for the schema generator to output dependencies on the same level grouped by the source file that contains the entity.

(I'm actually having a hard time even understanding how the above impls worked at all.)

I don't particularly care if some fallout from this PR is that the schema outputs in a different (but still valid, of course) order, but it would be nice if the output can maintain this property of same-file-entities are kept together.  

Sooo... I'm not seeing how instead deriving `Eq, Ord, PartialEq, PartialOrd, Hash` does that?